### PR TITLE
Add the login section in the GUI to login

### PR DESCRIPTION
**Pull request by Celthi**
@takashiro 
This code is to show how the front GUI to interact with the back end login API in: https://github.com/takashiro/asmodee-werewolf-server/pull/3
I also rename index.htm to index.html because .html may be more industrial and the root reason is that index.htm does not work in my nginx env.
This code is not the finalize yet, since I have add any CSS style and the logic to handle after the user login.

### CODE REVIEW

**Comment by Celthi** [comment]
Travis fails with 
error: Server does not allow request for unadvertised object 5d49ce71f27530d28dafad24e65360b933a46ec7
Fetched in submodule path 'src/game', but it did not contain 5d49ce71f27530d28dafad24e65360b933a46ec7. Direct fetching of that commit failed.

The command "eval git submodule update --init --recursive " failed. Retrying, 2 of 3.[0m

Is it the setting of the project?

File: src/gui/User/User.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+export default class User extends Component {
+	constructor(props) {
+		super(props);
+		this.state = {
+			name: 'guest'
+		}
+		this.onUpdatename = this.onUpdatename.bind(this);
+	}
+	onUpdatename(name) {
```
Comment (takashiro) [inline src/gui/User/User.js]: Camel-case naming?

File: src/gui/User/Login.scss [hunk 2]
```diff
@@ -0,0 +1,6 @@
+.guest {
+	visibility: visible;
```
Comment (takashiro) [inline src/gui/User/Login.scss:2]: Why not display: none? Do we need to reserve some space for the invisible element?

File: src/gui/User/login.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+			}
+			).catch((error) => {
+				alert(error);
+			});
+	}
+
+	render() {
+		return (<div className={this.state.status}>
+			<div>
+				<form onSubmit={this.login}>
```
Comment (takashiro) [inline src/gui/User/login.js]: This part needs to be styled.

File: src/index.js [hunk 4]
```diff
@@ -3,9 +3,13 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 
 import Lobby from './gui/Lobby';
-
+import User from './gui/User/User';
```
Comment (takashiro) [inline src/index.js]: The path /User/User here looks not hierarchical.

File: dist/index.html [hunk 5]
```diff
@@ -1,18 +1,22 @@
...
+	<meta name="apple-mobile-web-app-status-bar-style" content="black">
+	<meta name="author" content="Kazuichi Takashiro">
+	<link rel="shortcut icon" href="style/favicon.png">
+	<link rel="stylesheet" href="static/app.css">
+	<script src="static/vendor.js"></script>
 </head>
 
 <body>
+	<section>
+		<div id="user"></div>
```
Comment (takashiro) [inline dist/index.html:18]: It seems unnecessary to add a new root node. The implementation of login page is straightforward but not ideal.

File: src/gui/User/login.js [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+					<input type="text" id="id"></input>
+					<input type="text" id="name"></input>
+					<input type="submit"></input>
```
Comment (takashiro) [inline src/gui/User/login.js]: button tag is easier to customize styles.

**Review by takashiro — CHANGES_REQUESTED** [review]

**Comment by takashiro** [comment]
nginx is able to customize the root index file, as the nginx.conf you created contains 
```
index    index.html index.htm index.php;
```

**Comment by Celthi** [comment]
@takashiro 
Your comments are very high value. I'll change them when I have time.
Currently, the code is to demo the idea of the user login in the `asmodee-server`.
If we agree to use this approach, then I'll revise the code.
And sorry I missed you comment for a long time. I got a lot of messages by subscription a lot of project with no purpose.

File: src/gui/User/User.js (hunk 1 above)
Comment (Celthi) [reply src/gui/User/User.js]: Updated.

File: src/gui/User/Login.scss (hunk 2 above)
Comment (Celthi) [reply src/gui/User/Login.scss:2]: Changed.

File: src/gui/User/login.js (hunk 3 above)
Comment (Celthi) [reply src/gui/User/login.js]: Styled.

File: src/index.js (hunk 4 above)
Comment (Celthi) [reply src/index.js]: Changed.

File: src/gui/User/login.js (hunk 6 above)
Comment (Celthi) [reply src/gui/User/login.js]: Changed.

File: dist/index.html (hunk 5 above)
Comment (Celthi) [reply dist/index.html:18]: Yeah. But currently, I don't know how to structure it yet. ):

**Comment by Celthi** [comment]
Close this immature PR.

**Pull request closed without merging**